.env.example

```
@@ -1,79 +1,25 @@
-RAZORPAY_KEY_SECRET=""
-# Optional webhook verification for payment.captured
-# RAZORPAY_WEBHOOK_SECRET=""
-# Amount in smallest currency unit (default 99900 = ₹999.00)
-# RAZORPAY_PRO_AMOUNT="99900"
-# RAZORPAY_CURRENCY="INR"
-# Label shown on the pricing page
-# NEXT_PUBLIC_PRO_PRICE_LABEL="$10 / month"
-# How long a paid upgrade lasts (days). Coupon durationDays can override.
-# PRO_DURATION_DAYS="30"
+# Razorpay (fill with your keys)
+RAZORPAY_KEY_ID="************************"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)